datalad_service/tasks/description.py

```diff
@@ -15,7 +15,9 @@ def update_description(store, dataset, description_fields):
     description = ds.repo.repo.git.show(
         'HEAD:dataset_description.json')
     description_json = json.loads(description)
-    if any(description_fields):
+    if (not 'License' in description_json) or (description_json['License'] != 'CC0'):
```

Comment:
Just as a heads up, you can use [`dict.get`](https://docs.python.org/3/library/stdtypes.html#dict.get), which will do this check internally and return `None` if not found:

```suggestion
    if description_json.get('License') != 'CC0':
```